worker/obuilder_build.ml

```diff
@@ -6,7 +6,7 @@ type builder = Builder : (module Obuilder.BUILDER with type t = 'a) * 'a -> buil
 
 module Config = struct
   type t = {
-    store_spec : [ `Btrfs of string | `Rsync of string | `Zfs of string ];
+    store_spec : Obuilder.Store_spec.store Lwt.t;
```

Comment:
Does this need to be in Lwt.t see comment https://github.com/ocurrent/obuilder/pull/119/files#r974273224 ?
Perhaps it can just be `Obuilder.Store_spec.store` or does it need to re-run some effect each time?